tools/lint/lint.py

```diff
@@ -970,6 +982,7 @@ def create_parser():
                         help="Path to GitHub checks output file for Taskcluster runs")
     parser.add_argument("-j", "--jobs", type=int, default=0,
                         help="Level to parallelism to use (defaults to 0, which detects the number of CPUs)")
+    parser.add_argument("--rspfile", help="File containing a list of files to lint, one per line")
```

Comment:
I hadn't heard of the term "rspfile" before, but it seems to be somethng quite specific to MSBuild which defines command line switches? This doesn't seem quite the same as that because the file can't contain any random command line switches, but only ones relating to the path. Given that it's a fairly unusual term and that we aren't doing quite the same thing, I think we should rename this to something like `--include-file` or `--paths-file`.